Gemfile

```diff
@@ -23,6 +23,7 @@ manageiq_plugin "manageiq-content"
 manageiq_plugin "manageiq-providers-amazon"
 manageiq_plugin "manageiq-providers-azure"
 manageiq_plugin "manageiq-providers-hawkular"
+manageiq_plugin "manageiq-providers-google"
```

Comment:
If you point that to your extracted repo instead of the non existent manageiq repo, then travis would hopefully turn green. 
Later you can revert the commit and point it to the manageiq repo again